GPy/core/parameterization/priors.py

```diff
@@ -226,11 +233,11 @@ def pdf(self, x):
 
     def lnpdf(self, x):
         d = x - self.mu
-        return self.constant - 0.5 * np.sum(d * np.dot(d, self.inv), 1)
+        return self.constant - 0.5 * np.dot(d.T, np.dot(self.inv, d))
```

Comment:
This implementation does not work when x has more than one dimension.

Reply:
Thanks for taking a look. I think you're right, but is the multivariate Gaussian prior not meant for vectors of hyperparameters anyway? If x is a vector it works, and I don't see why someone would want to put a multivariate Gaussian prior on a matrix of hyperparameters